app/services/legacy/dataset_import_service.rb

```diff
@@ -127,7 +127,9 @@ def parse_date(date)
   end
 
   def datafile_name(resource)
-    resource['description'].strip == '' ? 'No name specified' : resource['description']
+    name = resource.fetch('name', '').strip
```

Comment:
http://api.rubyonrails.org/classes/String.html#method-i-squish might be a good option in this instance